### PR TITLE
ET-3836 Redirecting xayn.support to attlasian

### PR DESCRIPTION
**Pull request by jorge-cr-13**
## What 🕵️ 🔍

- What does the pull request add and/or solve?

## References 📝 🔗

- [Jira task](https://xainag.atlassian.net/browse/)

## Reviewer checklist

*Reviewer agreement:*

* Reviewers shouldn't review a "Draft Pull Request", but rather wait for it to be marked as "Ready for Review".
* Reviewers do **not** commit or merge the pull request.
* If/when there are changes that need to be applied (applying Terraform or deploying Kubernetes manifests), the **assignee** is the one responsible for doing it.
* Use the labels `not-applied`, `partially-applied` and `fully-applied` so others know the current progress.
  * Remember to change labels accordingly once you start and finish to apply the changes.

### CODE REVIEW

File: domains/.terraform-version [hunk 1]
```diff
@@ -0,0 +1 @@
+latest:^0.13
```
Comment (jorge-cr-13) [inline domains/.terraform-version:1]: We don't need this file.

File: domains/backend.tf [hunk 2]
```diff
@@ -3,5 +3,6 @@ terraform {
     bucket  = "terraform-state-xain"
     key     = "domains/terraform.tfstate"
     region  = "eu-central-1"
+    profile = "xayn-production"
```
Comment (jorge-cr-13) [inline domains/backend.tf:6]: This line is not necessary.

File: domains/providers.tf [hunk 3]
```diff
@@ -8,10 +8,12 @@ terraform {
 }
 provider "aws" {
   region  = "eu-central-1"
+  profile = "xain-production"
```
Comment (jorge-cr-13) [inline domains/providers.tf:11]: Same here

File: domains/providers.tf [hunk 4]
```diff
@@ -8,10 +8,12 @@ terraform {
...
 }
 
 
 provider "aws" {
   alias   = "us-east-1"
   region  = "us-east-1"
+  profile = "xain-production"
```
Comment (jorge-cr-13) [inline domains/providers.tf:18]: And here

File: internal_tools/redirection/main.tf [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+  source = "github.com/xaynetwork/xayn_terraform_modules//internal/redirection?ref=v0.5.0"
+
+  domain_name    = "support.xayn.com"
+  hosted_zone_id = data.terraform_remote_state.domain.outputs.xayn_primary_hosted_zone_id
+  host_name      = "xainag.atlassian.net/servicedesk/customer/portals"
+
+  # Optional
+  tags = {
+    Environment = "production"
+    User        = "provider"
```
Comment (jorge-cr-13) [inline internal_tools/redirection/main.tf:21]:
```suggestion
    Environment = "production"
    User        = "provider"
```
Do we need any of this?

**Review by Robert-Steiner — APPROVED** [review]

**Pull request merged**